cmd/dlt/userrole/cmd.go

```diff
@@ -151,6 +150,15 @@ func run(cmd *cobra.Command, argv []string) {
 		os.Exit(1)
 	}
 
+	roleExistOnAWS, existingRoleARN, err := r.AWSClient.CheckRoleExists(roleName)
+	if err != nil {
+		r.Reporter.Errorf("%v", err)
+	}
+	if !roleExistOnAWS || existingRoleARN != roleARN {
```

Comment:
I think we need here the separation between the cases like we have in `delete ocm role`.

Reply:
done